src/plone/importexport/utils.py

```diff
@@ -19,10 +21,10 @@ def __init__(self):
         self.in_memory_zip = StringIO.StringIO()
 
     def append(self, filename_in_zip, file_contents):
-        '''Appends a file with name filename_in_zip and contents of
-        file_contents to the in-memory zip.'''
+        """Appends a file with name filename_in_zip and contents of
+        file_contents to the in-memory zip."""
         # Get a handle to the in-memory zip in append mode
-        zf = zipfile.ZipFile(self.in_memory_zip, "a",
+        zf = zipfile.ZipFile(self.in_memory_zip, 'a',
```

Comment:
please break directly after the opening bracket and put each param in a single line, it is better to read